spec/syntax.md

```diff
@@ -470,6 +471,7 @@ Additionally, an _input-declaration_ can contain a _variable-expression_.
 > This placeholder contains an {|expression with a literal|}
 > This placeholder references a {$variable}
 > This placeholder references a function on a variable: {$variable :function with=options}
+> This placeholder has an option with a variable: {:function option=$variable}
```

Comment:
I think the first two examples here are overly cute.
```suggestion
> This placeholder contains a literal expression: {|literal|}
> This placeholder contains a variable expression: {$variable}
> This placeholder references a function on a variable: {$variable :function with=options}
> This placeholder contains a function expression with a variable-valued option: {:function option=$variable}
```

Reply:
... but I _like_ cute examples. 🐱  🐈 😺 

Oh, all right... no fun in the specification...